pkg/cmd/roachtest/tests/liquibase.go

```diff
@@ -99,13 +100,23 @@ func registerLiquibase(r registry.Registry) {
 		}
 
 		t.Status("running liquibase test harness")
-		// All tests are expected to pass, so this should not error.
-		// The dbVersion is set to 20.2 since that causes all known passing tests
-		// to be run.
-		if err = c.RunE(ctx, node,
+
+		rawResults, err := c.RunWithBuffer(ctx, t.L(), node,
 			`cd /mnt/data1/liquibase-test-harness/ && mvn test -Dtest=LiquibaseHarnessSuiteTest -DdbName=cockroachdb -DdbVersion=20.2`,
-		); err != nil {
-			t.Fatal(err)
+		)
+		rawResultsStr := string(rawResults)
```

Comment:
since this is a java test, i think we can do a better supported parsing approach using blocklists and stuff. like we just call `parseAndSummarizeJavaORMTestsResults` (see hibernate for example).

oh also, this failure is only affecting v21.2-alpha right? so maybe we need a check so if we backport this logic doesn't get applied.

Reply:
I used surefire but not sure if it was necessary to get the right output.

Updated to use `parseAndSummarizeJavaORMTestsResults`